### DISCUSSION TITLE
DAX query

### DISCUSSION THREAD

**Discussion by DAXQuestions** [DaxStudio/DaxStudio#871] [Q&A]
I am running a query utilizing: $SYSTEM.DISCOVER_CALC_DEPENDENCY  and would like to know what is the difference between Expression and Referenced Expression?  Additionally, if my table and reference table are the same in the query what does that indicate?  Lastly, if anyone know of a data dictionary available to provide context to what all the columns represent that would be very helpful.

**Comment by dgosbell** [comment]
> what is the difference between Expression and Referenced Expression

Expression is the text of the expression for the item in the OBJECT column while Referenced Expression relates to the item in the REFERENCEDOBJECT column. 

There is a data dictionary of all the columns for this DMV here https://docs.microsoft.com/en-us/openspecs/sql_server_protocols/ms-ssas/92efd9a8-8a36-4af7-9a95-3ff0810ed343 (and you can access all the other DMVs from the navigation menu on the right.
